__test__/integration/loan_offer/runners/accept.ts

```diff
@@ -0,0 +1,95 @@
+import * as Web3 from "web3";
+
+import { BigNumber, Dharma } from "../../../../src";
+
+import { LoanOffer } from "../../../../src/types";
+
+import { ContractsAPI } from "../../../../src/apis/";
+
+import { DummyTokenContract } from "../../../../src/wrappers";
+
+// Accounts
+import { ACCOUNTS } from "../../../accounts";
+
+const { Token } = Dharma.Types;
+
+const CREDITOR = ACCOUNTS[0];
+const DEBTOR = ACCOUNTS[1];
+
+const TX_DEFAULTS = { from: CREDITOR.address, gas: 4712388 };
+
+export async function testAccept(
+    dharma: Dharma,
+    params: any,
+    signAndAccept: (loanOffer: LoanOffer, address: string) => void,
+) {
+    describe("passing valid params", () => {
+        let loanOffer: LoanOffer;
+
+        let web3: Web3;
+
+        let contractsApi: ContractsAPI;
+
+        let principalToken: DummyTokenContract;
```

Comment:
we can remove a lot of this setup code by using a shared util function. see below.

Reply:
done.